### PR TITLE
feat: check if nft exists once selected for transfer

### PR DESCRIPTION
**Pull request by JackHamer09**

### CODE REVIEW

File: src/blocks/Transaction.vue [hunk 1]
```diff
@@ -148,7 +152,7 @@
         <div class="_display-flex _justify-content-center _align-items-center">
           <v-icon v-if="!hasSigner && requireSigner" name="md-vpnkey-round" />&nbsp;&nbsp;
           <div>{{ hasSigner || !requireSigner ? "" : "Authorize to " }}{{ transactionActionName }}</div>
-          <loader v-if="allowanceLoading || buttonLoader || requestingSigner" class="_margin-left-1" size="xs" />
+          <loader v-if="allowanceLoading || (!nftExists && nftExistsLoading) || buttonLoader || requestingSigner" class="_margin-left-1" size="xs" />
```
Comment (keriat) [inline src/blocks/Transaction.vue:155]: Move it to the computed property plz

Comment (keriat) [reply src/blocks/Transaction.vue:155]: Single one I mean

**Review by keriat — CHANGES_REQUESTED** [review]
Slight change requested

File: src/blocks/Transaction.vue [hunk 2]
```diff
@@ -134,6 +134,10 @@
       <div v-if="type === 'CPK' && cpkStatus === true" class="_text-center _margin-top-1">Your account is already activated</div>
 
       <div v-if="error" class="errorText _text-center _margin-top-1" data-cy="transaction_error_text">{{ error }}</div>
+      <div v-if="chosenToken && mainToken === 'L2-NFT' && !nftExists && !nftExistsLoading" class="errorText _text-center _margin-top-1">
```
Comment (keriat) [inline src/blocks/Transaction.vue:137]: Implode it into computed property. This should be done any time you're using more then 2 conditions directly in the code

File: src/blocks/Transaction.vue [hunk 3]
```diff
@@ -386,10 +399,12 @@ export default Vue.extend({
     },
   },
   async mounted() {
+    if (!this.$store.getters["zk-account/loggedIn"]) {
```
Comment (keriat) [inline src/blocks/Transaction.vue:402]: Shouldn't this check be moved to the middleware?

**Review by keriat — CHANGES_REQUESTED** [review]
Minor changes requested

**Review by AnastasiiaVashchuk — APPROVED** [review]

**Review by keriat — APPROVED** [review]
LGTM

**Pull request merged**